packages/office-ui-fabric-react/__mocks__/@uifabric/utilities.ts

```diff
@@ -0,0 +1,33 @@
+export * from '@uifabric/utilities';
+import { Async } from '@uifabric/utilities';
+
+// Known issue with jest's runAllTimers and debounce implementations resulting in
+// "Ran 100000 timers, and there are still more! Assuming we've hit an infinite recursion and bailing out..."
+// https://github.com/facebook/jest/issues/3465
+// Mock impl inspired from issue.
+class MockAsync extends Async {
+  public debounce(callback: Function, timeout: number) {
+    let timeoutId: number | null = null;
+    const debounced = (...args: any[]) => {
+      if (timeoutId) {
+        clearTimeout(timeoutId);
```

Comment:
Shouldn't you also do `timeoutId = null` both here and in `cancel`?

Reply:
Well, it gets set immediately again below, and would have no effect in timeoutId, and it's a mock. But I can add them just to be safe.

Reply:
Yeah it's more an issue of general good patterns than an actual problem in this case.